### ISSUE TITLE
Define valid values of a set of options for an option

### ISSUE THREAD

**Issue by isometriq** [docopt/docopt.cpp#86]
Fiven the following usage string:
```
Usage:
  vrplayer-openvr [--option VALUE]

Options:
  --option VALUE  Set of options (first|second|third) [default: first]
```
Is it possible to validate a set of options (see the values in the parenthesis.

Could it be written also as:
```
  --param VALUE  Set of options [values: first | second | third, default: first]
```

If not possible, then its a feature request.

**Comment by isometriq** [comment]
What is expected is that the parser also restrict the used option value (and throws if invalid).
In the my example, I have a pipe list, but actually could it be a regex?

**Comment by jaredgrubb** [comment]
No, the syntax doesn't have any way to do this .. it would be up to you to provide that kind of validation when you look at the values you get. I think trying to adjust the syntax would be tricky to do.

**Comment by piotr-dobrogost** [comment]
What about syntax proposed above by @isometriq ?
`  --option VALUE  Description… [values: first|second|third, default: first]`
